media/transcoder/src/test/java/com/example/transcoder/CreateJobFromAdHocTest.java

```diff
@@ -122,7 +122,7 @@ public void test_CreateJobFromAdHoc() throws Exception {
     JOB_ID = arr[arr.length - 1].replace("\n", "");
     bout.reset();
 
-    Thread.sleep(180000);
+    Thread.sleep(300000);
```

Comment:
Is there a reason we're updating the timeout for these tests in this PR? Might be better to update timeouts for other tests in a separate PR.

Reply:
Not sure why this is showing up. This was already updated in https://github.com/GoogleCloudPlatform/java-docs-samples/commit/8ebe2b9896ccae10205189786b1bf8b905f7fd39